### PR TITLE
[kaggle_project] Laura Villalba

### PR DESCRIPTION
**Pull request by lauravillalba**

### CODE REVIEW

**Comment by ferrero-felipe** [comment]
Muy buen trabajo, Laura!
Super organizados, ambos el codigo y el repositorio. :top:
Te comento apenas unas cositas.
Si creas un modulo con la funcion de limpieza de el dataset, basta importarla y no hace falta repetirle en todos los `jupyter`.
O, mejor, si creas un  `jupyter` que sea solo para limpiar y preparar los datos, puedes exportar el csv de los datos limpios y no hace falta repetir el mismo procedimiento en todos los demás y lo haces una unica vez.
La otra cosa es que el `train_test_split` no hace siempre la misma division de los datos porque utiliza de aleatoriedad. Si quieres probar los diferentes modelos con la misma division de los datos para que la comparacion sea más verdadera, tienes que dividirles solo una vez o definir el parametro `random_state` al llamar `train_test_split` para que esta funcion use siempre el mismo `seed` de pseudo aleatoriedad.
Pero ya te digo, muy buen trabajo con el machine learning! :rocket:

¡Sigue Así!

image
